pkg/upgrade/plan/plan.go

```diff
@@ -239,7 +241,7 @@ func sha256sum(s ...string) string {
 }
 
 // Validate performs validation of the plan spec, raising errors for any conflicting or invalid settings.
-func Validate(plan *upgradeapiv1.Plan) error {
+func Validate(plan *upgradeapiv1.Plan, secretCache corectlv1.SecretCache) error {
```

Comment:
is this a nit on correct (sp?)

Reply:
nit on the spelling of what, the package? That is how its always been imported in this file:
https://github.com/rancher/system-upgrade-controller/blob/9de913246abe4e1d6fb1f04939c5b805a436a44b/pkg/upgrade/plan/plan.go#L20